### ISSUE TITLE
Just kindly asking for the pretrained weights download link

### ISSUE THREAD

**Issue by RorschachChen** [jiangw-0/LE_JCDP#1]
Also, wondering which DDPM pretrained weights(from which repo) you use, for purify CIFAR10. Thanks a lot.

**Comment by RorschachChen** [comment]
Just being approved for access to the pretrained model. Thanks.

**Comment by jiangw-0** [comment]
Sorry, I forgot to change permissions when uploading. Since the assumption of our work is that clean samples of unlearnable samples cannot be obtained, the DDPM of cifar10 is trained from scratch using cifar100 through the [ddim](https://github.com/ermongroup/ddim) repo, and then fine-tuned on the test set of cifar10.

**Comment by RorschachChen** [comment]
Perfect solved my question. Thanks.
